src/composables/useOpenapi.ts

```
@@ -2,6 +2,8 @@
 import type { OpenAPIDocument } from '../types'
 import type { PartialUseThemeConfig } from './useTheme'
 import { createOpenApiInstance } from '../lib/createOpenApiInstance'
+import { prepareOpenAPI } from '../lib/prepareOpenAPI/prepareOpenAPI'
```

ESLint

[error] 'prepareOpenAPI' is defined but never used.

(unused-imports/no-unused-imports)